db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,81 @@
+class AddCommentsToVmsTable < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "Indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "ID of the availability zone that the VM is part of."
+    change_column_comment :vms, :busy, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "Indicates whether or not the VM/Template is cloud or infra."
```

Comment:
We really have a column for this?  I don't see amazon setting this, need to see if anyone else does.

Reply:
We set it in Azure, though I don't remember how it's used, if at all.

Reply:
I think we had added it for the UI so they could more easily differentiate.  We'd have to go into the history otherwise.

Reply:
@agrare @Fryguy Should I mark this deprecated, or update it in some way? Otherwise, please resolve.

Reply:
I don't see any of the cloud parsers setting this but lets hold off on marking it deprecated